itl/plugins-contrib.d/systemd.conf

```diff
@@ -1,51 +1,99 @@
+		}
+		"-vvv" = {
+			set_if = {{ macro("$systemd_verbose_level$") == 3 }}
+		}
+
+		/* Options related to unit selection */
+		"--ignore-inactive-state" = {
+			value = "$systemd_ignore_inactive_state$"
+			description = "Ignore an inactive state on a specific unit. Only affective if used with `systemd_unit`."
+		}
+		"--include" = {
+			value = "$systemd_include$"
+			description = "Include systemd units to the checks, regular expressions are supported. This option can be applied multiple times."
+			repeat_key = true
+		}
 		"--unit" = {
 			value = "$systemd_unit$"
 			description = "Name of the systemd unit that is being tested."
 		}
+		"--include-type" = {
+			value = "$systemd_include_type$"
+			description = "Unit types to be tested (for example: `service`, `timer`). This option can be applied multiple times."
+			repeat_key = true
+		}
 		"--exclude" = {
 			value = "$systemd_exclude_unit$"
-			description = "Exclude a systemd unit from the checks. This option can be applied multiple times. Also supports regular expressions."
+			description = "Exclude a systemd unit from the checks, regular expressions are supported. This option can be applied multiple times."
 			repeat_key = true
 		}
-		"--no-startup-time" = {
-			set_if = "$systemd_no_startup_time$"
-			description = "Don’t check the startup time. Using this option the options `systemd_warning` and `systemd_critical` have no effect. (Default: `false`)"
+		"--exclude-unit" = {
+			value = "$systemd_exclude_unit_name$"
+			description = "Exclude a systemd unit from the checks. This option can be applied multiple times."
+			repeat_key = true
 		}
-		"--warning" = {
-			value = "$systemd_warning$"
-			description = "Startup time in seconds to result in a warning status. (Default: `60s`)"
+		"--exclude-type" = {
+			value = "$systemd_exclude_type$"
+			description = "Exclude a systemd unit type (for example: `service`, `timer`)"
 		}
-		"--critical" = {
-			value = "$systemd_critical$"
-			description = "Startup time in seconds to result in a critical status. (Default: `120s`)"
+		"--state" = {
+			value = "$systemd_state$"
+			description = "Specify the active state that the systemd unit must have (for example: `active`, `inactive`)"
 		}
+
+		/* Timers related options */
 		"--dead-timers" = {
-			set_if = "$systemd_dead_timers$"
-			description = "Detect dead / inactive timers. (Default: `false`)"
+			value = "$systemd_dead_timers$"
+			description = "Detect dead / inactive timers, see `systemd_dead_timers_{warning,critical}`. (Default `false`)"
 		}
-		"--dead-timers-warning" = {
+		"--timers-warning" = {
 			value = "$systemd_dead_timers_warning$"
-			description = "Time ago in seconds for dead / inactive timers to trigger a warning state (by default 6 days)."
+			description = "Time ago in seconds for dead / inactive timers to trigger a warning state. (Default 6 days)"
 		}
-		"--dead-timers-critical" = {
+		"--timers-critical" = {
```

Comment:
Removing the `dead` prefix while still referring to **dead** timers and not just timers doesn't make sense to me, so I wouldn't change this.